tests/foreman/cli/test_computeresource_azurerm.py

```diff
@@ -0,0 +1,530 @@
+                          deleted in AzureRm CR along with username
+
+        :CaseImportance: Critical
+
+        :CaseLevel: Integration
+        """
+
+        # Create
+        img_name = gen_string('alpha')
+        username = gen_string('alpha')
+        img_ft = ComputeResource.image_create(
+            {
+                'name': img_name,
+                'operatingsystem-id': module_os.id,
+                'architecture-id': module_architecture.id,
+                'uuid': image,
+                'compute-resource': module_azurerm_cr.name,
+                'username': username,
+                'user-data': 'no',
+            }
+        )[0]
+        assert img_ft['message'] == 'Image created.'
+        assert img_ft['name'] == img_name
+
+        # Info Image
+        img_info = ComputeResource.image_info(
+            {'name': img_name, 'compute-resource': module_azurerm_cr.name}
+        )[0]
+        assert img_info['operating-system'] == module_os.title
+        assert img_info['username'] == username
+        assert img_info['uuid'] == image
+        assert img_info['user-data'] == 'false'
+        assert img_info['architecture'] == module_architecture.name
+
+        # List image
+        list_img = ComputeResource.image_list({'compute-resource': module_azurerm_cr.name})
+        assert len(list_img) == 1
+        assert list_img[0]['name'] == img_name
+
+        # Update image
+        new_img_name = gen_string('alpha')
+        new_username = gen_string('alpha')
+        result = ComputeResource.image_update(
+            {
+                'name': img_name,
+                'compute-resource': module_azurerm_cr.name,
+                'new-name': new_img_name,
+                'username': new_username,
+            }
+        )[0]
+        assert result['message'] == 'Image updated.'
+        assert result['name'] == new_img_name
+
+        img_info = ComputeResource.image_info(
+            {'name': new_img_name, 'compute-resource': module_azurerm_cr.name}
+        )[0]
+        assert img_info['username'] == new_username
+
+        # Delete Image
+        result = ComputeResource.image_delete(
+            {'name': new_img_name, 'compute-resource': module_azurerm_cr.name}
+        )[0]
+        assert result['message'] == 'Image deleted.'
+        assert result['name'] == new_img_name
+
+    @tier2
+    def test_positive_check_available_networks(self, azurermclient, module_azurerm_cr):
+        """Check networks from AzureRm CR are available to select during host provision.
+
+        :id: 9e08463c-c700-47fc-8a58-e03aa8bcd097
+
+        :expectedresults: All the networks from AzureRM CR should be available.
+
+        :CaseLevel: Integration
+        """
+
+        result = ComputeResource.networks({'id': module_azurerm_cr.id})
+        assert len(result) > 0
+
+    @tier2
+    def test_positive_create_compute_profile_values(self, azurermclient, module_azurerm_cr):
```

Comment:
```suggestion
    def test_positive_create_compute_profile(self, azurermclient, module_azurerm_cr):
```

Reply:
tests is more related to `values` create instead profile create hence used same naming ..

Reply:
We always test values of any component as a componenet is made of values only. :)

But not a blocker comment to hold this PR. ACKing and Merging.